5 - RPG/Code/RPG.CharacterBuilder/CharacterAttributes.cs

```diff
@@ -0,0 +1,24 @@
+﻿using RPG.CharacterBuilder.CharacterAttributeClasses;
+
+namespace RPG.CharacterBuilder
+{
+    public class CharacterAttributes
+    {
+        public HeadRoundness Roundness;
+        public Height Height;
+        public FitnessLevel Fitness;   
```

Comment:
You have public *variables*, only properties should be public. I like you abstraction of your attributes so you have a container for them, but make sure to keep variables as private.